phoenix-spark/src/test/java/org/apache/phoenix/spark/datasource/v2/PhoenixDataSourceTest.java

```diff
@@ -64,6 +67,8 @@ public void testPhoenixConfigsExtractedProperly() {
         assertEquals(V1, p.getProperty(P1));
         assertEquals(V2, p.getProperty(P2));
         assertEquals(V3, p.getProperty(P3));
+        assertEquals(V3, p.getProperty(P3));
+        assertEquals(true, Boolean.valueOf(p.getProperty(NO_CACHE)));
```

Comment:
This test isn't testing your change at all..Here you are using the extraOptions to set the property and just checking that the property is set. Ideally, we want to use extraOptions to set HBase/Phoenix properties if they are valid configs we set in say hbase-site.xml. In this case, `NO_CACHE` is not such a config so we are using a different way to set this.